src/booleans.js

```diff
@@ -1,61 +1,88 @@
 const negate = (a) => {
-  // your code here
+  return 1 > a
 };
 
 const both = (a, b) => {
-  // your code here
+  return a > 0 && b > 0
 };
 
 const either = (a, b) => {
-  // your code here
+  return a > 0 || b > 0
 };
 
 const none = (a, b) => {
-  // your code here
+  if (a || b) {
+    return false
+  } else {
+    return true
+  }
```

Comment:
The `a || b` expression inside the `if` statement evaluates to `true` if either `a` or `b` is a truthy value, otherwise it will evaluate to false. Therefore, you only need to return the expression.